### PR TITLE
Split remediation schedules, UI controls, auto-bootstrap, and tests

### PR DESCRIPTION
**Pull request by jorgecuesta**
## Summary

- **Remediation schedule split**: Separate `SupplierRemediation` (1001 ServiceMismatch, pausable by operator) from `SupplierInitialStake` (1003, always active). Both use the same underlying workflow with different args.
- **Provider UI controls**: Auto Stake toggle (pause/unpause schedule), Request Remediation (two-step evaluate→confirm), renamed "Mark for Remediation" to "Clear Remediation"
- **Workflow fixes**: Owner address backfill for legacy keys, early return when no keys exist, fix Temporal client namespace defaults
- **Auto-bootstrap for dev**: Idempotent seed scripts for provider and middleman. Tilt conditionally injects init container from local JSON config. Fetches minimumStake/height from Pocket API and delegators/providers from governance CDN.
- **Tests**: 14 new tests with realistic fixtures covering service comparison edge cases from the re-staking loop bug
- **Docs & UI consistency**: Unified "Pocket API URL" label, updated DEVELOP.md with auto-bootstrap setup, updated key-management docs with new actions
- **Tilt improvements**: Simplified ignore patterns to prevent local test/build runs from triggering rebuild cycles

## Test plan

- [x] All domain tests pass locally (49 tests, 4 suites)
- [x] Tilt cluster boots cleanly with auto-bootstrap (both apps seed from JSON + governance CDN)
- [x] Provider and provider-workflows running with 0 restarts
- [x] Middleman and middleman-workflows running with 0 restarts
- [x] Bootstrap seed is idempotent (skips on re-deploy)
- [x] Workflows handle empty key set gracefully (no crash)
- [x] Auto Stake toggle pause/unpause works from UI
- [ ] Verify Request Remediation evaluate→confirm flow (needs staked keys — will test with localnet)
- [ ] Verify Clear Remediation button behavior (needs keys in error state)
- [ ] CI passes

### CODE REVIEW

**Review by TheFeloniousMonk — APPROVED** [review]

File: apps/provider/src/app/admin/(internal)/keys/AutoStakeToggle.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        setError(result.error.message)
+      }
+    }).catch((e) => {
+      if (cancelled) return
+      setError(e instanceof Error ? e.message : 'Failed to fetch schedule status')
+    }).finally(() => {
+      if (!cancelled) setIsLoading(false)
+    })
+    return () => { cancelled = true }
+  }, [])
```
Comment (Alann27) [inline apps/provider/src/app/admin/(internal)/keys/AutoStakeToggle.tsx:33]: Use react-query `useQuery` for this

File: apps/provider/src/actions/Remediation.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      } else {
+        history.push(entry)
+        history.sort((a, b) => b.timestamp - a.timestamp)
+      }
+
+      updates.push({
+        address: key.address,
+        remediationHistory: history,
+      })
+    }
```
Comment (Alann27) [inline apps/provider/src/actions/Remediation.ts:113]: should we check that the key is having a service mismatch here?

File: apps/provider/src/app/admin/(internal)/keys/AutoStakeToggle.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  const handleConfirm = async () => {
+    setError(null)
+    setIsSubmitting(true)
+    try {
+      const result = await ToggleRemediationSchedule(!paused)
+      if (!result.success) {
+        setError(result.error.message)
+        return
+      }
+      setPaused(!paused)
```
Comment (Alann27) [inline apps/provider/src/app/admin/(internal)/keys/AutoStakeToggle.tsx:50]: instead of using the hook directly to set the new value we should do `setPaused((prevPaused) => !prevPaused)`

File: apps/provider/src/lib/dal/keys.ts [hunk 4]
```diff
@@ -275,3 +277,131 @@ export async function updateKeysStateWhereCurrentStateIn(currentStates: KeyState
...
+  const dbClient = getDbClient()
+  await Promise.all(
+    updates.map((update) =>
+      dbClient.db
+        .update(keysTable)
+        .set({ remediationHistory: update.remediationHistory })
+        .where(eq(keysTable.address, update.address)),
+    ),
+  )
+}
```
Comment (Alann27) [inline apps/provider/src/lib/dal/keys.ts:353]: we should use a transaction here
